src/main/java/lotto/domain/WinningNumbers.java

```diff
@@ -18,18 +19,27 @@ public WinningNumbers(Set<Integer> winningNumbers) {
 			throw new IllegalArgumentException("당첨 번호는 " + LOTTO_NUMBER_MIN + " 미만 " + LOTTO_NUMBER_MAX + " 초과인 수를 입력할 수 없습니다.");
 		}
 
+		if(bonusNumber < LOTTO_NUMBER_MIN || bonusNumber > LOTTO_NUMBER_MAX) {
```

Comment:
검증 잘 해주셨습니다 👍 
```suggestion
		if (bonusNumber < LOTTO_NUMBER_MIN || bonusNumber > LOTTO_NUMBER_MAX) {
```

근데 로또 번호 (1~45)는 당첨번호에만 검증이 필요한게 아니라 수동로또에도 필요한 부분인데요 😄 
이 부분은 어떻게 할 지 마지막 단계 진행하시면서 고민해주시면 될 것 같습니다 🙇